1-mandatory/2-people-i-know.js

```diff
@@ -376,32 +376,68 @@ const friends = [
 In the above object you can see my friends and the colleagues of my friends.
 First, I want you to find all of my friends who are 35 or older.
 */
-
-let thirtyFiveOrOlder = [];
+let thirtyFiveOrOlder = friends.filter((friend) => friend.age >= 35);
 
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
-let powerNetEmails = [];
+  let powerNetEmails = friends
+  .filter((friend) => friend.company.includes("POWERNET"))
+  .map((friend) => friend.email);
+
 
+  
+ 
 /*
 4) colleagues with "Stacie Villarreal"
 Next, I want you to find all of my friends who are colleagues of Stacie Villarreal.
 You can see who people's colleagues are by seeing the "colleagues" array in each of my friends objects.
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
```

Comment:
This is good. Maybe the `some` array method would be better to use than `find` here.
Both will work, but if you use `some` - other developers who are reading your code can tell that you are only interested in whether the friend has a colleague that knows Stacie, and you're not interested in who that colleague is. This is about how we can let our code express our intention 😄